app/states/unified_sqlmodels.py

```
@@ -0,0 +1,1477 @@
+        """Normalize a field name for comparison"""
+        if field_name is None:
+            return ""
+        try:
+            normalized = str(field_name).lower()
+            normalized = re.sub(r'[^a-z0-9]', '_', normalized)
+            normalized = re.sub(r'_+', '_', normalized)
+            return normalized.strip('_')
+        except Exception:
+            return ""
+    
+    def _parse_amount(self, value: Any) -> Optional[Decimal]:
```

Ruff (0.14.7)

Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)